packages/atlas-anatomogram-experiment-table/src/anatomogramExperimentTable.js

```diff
@@ -0,0 +1,313 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Anatomogram from '@ebi-gene-expression-group/organ-anatomogram'
+import { TableManager } from '@ebi-gene-expression-group/atlas-experiment-table'
+import URI from 'urijs'
+
+const capitalizeFirstLetter = (str) => str.charAt(0).toUpperCase() + str.slice(1)
+
+const ontologyAccessionToOntologyUri = (accession) => {
+  return accession.startsWith(`EFO`) ?
+    `http://www.ebi.ac.uk/efo/${accession}` :
+    `http://purl.obolibrary.org/obo/${accession}`
+}
+
+
+class AnatomogramExperimentTable extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      selectedSpecies: props.species,
+      showIds: props.showIds,
+      experiments: props.experiments,
+      highlightIds: [],
+      selectIds: [],
+      selectAllIds: [],
+      selectedOrganId: ``
+    }
+
+    this._addRemoveFromSelectIds = this._addRemoveFromSelectIds.bind(this)
+    this._showLinkBoxIds = this._showLinkBoxIds.bind(this)
+    this._clearSelectedIds = this._clearSelectedIds.bind(this)
+    this._handleSelectOnChange = this._handleSelectOnChange.bind(this)
+    this.handleResponse = this.handleResponse.bind(this)
+    this._afterSwitchView = this._afterSwitchView.bind(this)
+    this.getExperimentsAndUpdateState = this.getExperimentsAndUpdateState.bind(this)
+  }
+
+  _handleSelectOnChange(event) {
+    event.persist();
+    this.setState(() => ({
+      selectedSpecies: event.target.value
+    }))
+  }
+
+  _addRemoveFromSelectIds(ids) {
+    if (ids.length === 0 || ids[0].startsWith(`link`)) {
+      this.setState({
+        selectIds: ids
+      })
+    } else {
+      this.setState({
+        selectIds: ids,
+        resource:
+          URI(`json/experiments/${this.props.experimentAccession}/marker-genes/cell-types`, this.props.host)
+            .search({ organismPart: ids.map(ontologyAccessionToOntologyUri).join(`,`) })
+            .toString()
+      })
+    }
+    this.getExperimentsAndUpdateState(this.props)

```

Comment:
Since `getExperimentsAndUpdateState` always uses the host from `props` and the `resource` from the state, I think we can modify it so it doesn’t take any paramteres at all.